src/main/java/github/scarsz/discordsrv/objects/threads/ChannelUpdater.java

```diff
@@ -0,0 +1,157 @@
+/*-
+ * LICENSE
+ * DiscordSRV
+ * -------------
+ * Copyright (C) 2016 - 2021 Austin "Scarsz" Shapiro
+ * -------------
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * END
+ */
+
+package github.scarsz.discordsrv.objects.threads;
+
+import github.scarsz.discordsrv.Debug;
+import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.util.DiscordUtil;
+import github.scarsz.discordsrv.util.PlaceholderUtil;
+import lombok.Getter;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.GuildChannel;
+import net.dv8tion.jda.api.exceptions.PermissionException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class ChannelUpdater extends Thread {
+
+    @Getter private final Set<UpdaterChannel> updaterChannels = new HashSet<>();
+
+    public ChannelUpdater() {
+        setName("DiscordSRV - Channel Updater");
+    }
+
+    public void reload() {
+        // Deleting and recreating the list of updater channels
+        this.updaterChannels.clear();
+
+        final List<Map<String, Object>> configEntries = DiscordSRV.config().getList("ChannelUpdater");
+        for (final Map<String, Object> configEntry : configEntries) {
+            final String channelId = configEntry.get("ChannelId").toString();
+            final String format = configEntry.get("Format").toString();
+            final String intervalAsString = configEntry.get("UpdateInterval").toString();
+            final int interval;
+
+            if (channelId.equals("0000000000000000")) continue; // Ignore default
+
+            if (StringUtils.isAnyBlank(channelId, format)) {
+                DiscordSRV.error("Failed to initialise a ChannelUpdater entry: Missing either ChannelId or Format");
```

Comment:
I don't think having this as a error is necessary, should be pretty obvious having them blank isn't allowed

Reply:
You never know really. I think there should at least be a warning or a debug message

Reply:
Being able to set it to "" in the config shouldn't cause a error

Reply:
Should we keep as a warning or debug?

Reply:
debug